includes/documents/class-wcpdf-packing-slip.php

```diff
@@ -146,7 +146,29 @@ public function init_settings() {
 				)
 			),
 		);
-
+		
+		if ( ! function_exists( 'WPO_WCPDF_Pro' ) ) {
```

Comment:
maybe a double check here whether `attach_to_email_ids` is already present?

Reply:
By present you mean saved? Because Pro adds it using the filter below this new setting: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/379-packing-slip-attach-to-notice/includes/documents/class-wcpdf-packing-slip.php#L174

Reply:
I mean that it's already in the array. But you're right, since `wpo_wcpdf_settings_fields_documents_packing_slip` is applied after this code, there's no point in checking!